src/foraging_gui/Foraging.py

```diff
@@ -2043,14 +2047,18 @@ def _Save(self,ForceSave=0,SaveAs=0,SaveContinue=0):
         Obj['PhotometryFolder']=self.PhotometryFolder
         Obj['MetadataFolder']=self.MetadataFolder
         
+        # only run once for each session
```

Comment:
This comment seems confusing now, since the logic to only check for dropped frames once is not here

Reply:
Removed!